### ISSUE TITLE
ReferenceError: window is not defined

### ISSUE THREAD

**Issue by joeduncan** [TroyAlford/react-jsx-parser#64]
```
ReferenceError: window is not defined
    at Object.<anonymous> (/node_modules/react-jsx-parser/lib/react-jsx-parser.min.js:1:285)
    at Module._compile (internal/modules/cjs/loader.js:702:30)
```
Happens when updating `react-jsx-parser  1.3.2  →  1.3.4`

There seems to be a new dependency to `window` - is it still possible to use the latest version on server-side?

**Comment by TroyAlford** [comment]
This seems related to the recent upgrade to Babel 7. I suspect it has something to do with the transpilation being slightly changed, but haven't been able to narrow it down.

For the moment, the build mechanism is the only difference between 1.3.3 and 1.3.4 - so I'd suggest upgrading to 1.3.3 instead. If you put together a minimalistic repro for me, I'd be happy to dig into this a bit more. It's unclear to me exactly what changed with the Babel 7 change to cause this.

**Comment by joeduncan** [comment]
1.3.3 works fine thanks!

here a short repro for 1.3.4:

```
import React from 'react';
import JSXParser from 'react-jsx-parser';

React.createElement(JSXParser, {
    bindings: {},
    jsx: '<div />'
});
```
`node --experimental-modules repro.mjs`

**Comment by joeduncan** [comment]
Seems to be a Webpack 4 issue: 
https://github.com/webpack/webpack/issues/6642
https://github.com/webpack/webpack/issues/6525

We could always provide different builds for different cases (es modules/umd/commonjs) - what do you think?
redux maybe a good example: https://github.com/reduxjs/redux/blob/master/package.json

**Comment by TroyAlford** [comment]
Yeah, I'd be ok with that. It's a bit beyond my current understanding of the issue at hand - but if you think you have a bead on it, I'd love a PR?

**Comment by TroyAlford** [comment]
Closed by #65. Fix released in 1.3.6
